main.js

```diff
@@ -250,15 +250,35 @@ window.onload = function() {
    toggle(this);
  };
  document.getElementById("clearCounters").onclick = function() {
-   J2ME.runtimeCounter && J2ME.runtimeCounter.clear();
-   J2ME.nativeCounter && J2ME.nativeCounter.clear();
-   J2ME.interpreterCounter && J2ME.interpreterCounter.clear();
-   J2ME.interpreterMethodCounter && J2ME.interpreterMethodCounter.clear();
-   J2ME.baselineMethodCounter && J2ME.baselineMethodCounter.clear();
-
+   clearCounters();
  };
+
+  setInterval(function () {
+    function numberWithCommas(x) {
+      return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+    }
```

Comment:
This function should get declared outside the setInterval call, so it doesn't have to be redeclared each time the interval function is called. But does this really need to get called unconditionally? Seems like it should be disabled by default.